include/openenclave/enclave.h

```diff
@@ -724,6 +724,61 @@ oe_enclave_t* oe_get_enclave(void);
  */
 oe_result_t oe_random(void* data, size_t size);
 
+/**
+ * oe_gen_tls_cert.
+ *
+ * This function generates a self-signed certificate with an embedded quote from
+ * the underlying enclave.
+ *
+ * @param issuer_key: a private key used to sign this certificate
+ * @param issuer_key_size The size of the issuer_key buffer.
+ * @param subject_key: a public key used to in the certificate
```

Comment:
can't parse "used to in".  Also a nit that some lines end in . and some don't

Reply:
fixed